vsts/scripts/publishSdksFromDevToProdStorageAccount.sh

```diff
@@ -4,13 +4,19 @@
 # Licensed under the MIT license.
 # --------------------------------------------------------------------------------------------
 
-set -ex
+set -e
```

Comment:
Curious why we're removing the `x`? The trace sometimes helps with debugging

Reply:
@william-msft I removed the x since in my experience with these jobs, the extra printing creates some noise in the job. Its a lot easier to review what is going to change during the dry-run (which we will be looking at before manually approving), and in this script we are quite verbose with our printing, to the point where we can quickly determine what step/blob it may have failed on.

Reply:
Sounds good